<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Inicio Sesion</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 203, 151);</string>
  </property>
  <widget class="QPushButton" name="pushButton_ingresar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 51, 102);
color: rgb(255, 255, 255);
border-radius: 5px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);</string>
   </property>
   <property name="text">
    <string>INGRESAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FONTSPRING DEMO - Organetto Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>USUARIO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FONTSPRING DEMO - Organetto Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CONTRASEÃ‘A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contrasena">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usuario">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_usuario</tabstop>
  <tabstop>lineEdit_contrasena</tabstop>
  <tabstop>pushButton_ingresar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
